CSS/lists.css

```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+.lists-container {
+    max-width: 400px;
+    margin: auto;
+}
+
+.ordered-list {
+    padding: 2rem;
+}
+
+.unordered-list {
+    padding: 2rem;
+}
```

Comment:
you can have a class for padding and use those classes wherever required. Else GTG